invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CopyrightsField/CopyrightsField.js

```diff
@@ -7,40 +7,69 @@
 // under the terms of the MIT License; see LICENSE file for more details.
 
 import React, { Component } from "react";
-import PropTypes from "prop-types";
-
-import { FieldLabel, TextField } from "react-invenio-forms";
+import Overridable from "react-overridable";
 import { i18next } from "@translations/invenio_rdm_records/i18next";
+import {
+  createCommonDepositFieldComponent,
+  fieldCommonProps,
+  FieldLabel,
+  TextField,
+} from "react-invenio-forms";
 
-export class CopyrightsField extends Component {
+class CopyrightsFieldComponent extends Component {
   render() {
-    const { fieldPath, label, required } = this.props;
+    const { fieldPath, label, labelIcon, required, disabled, helpText, placeholder } =
+      this.props;
+
     return (
-      <TextField
-        fieldPath={fieldPath}
-        label={
-          <FieldLabel htmlFor={fieldPath} icon="copyright outline" label={label} />
-        }
+      <Overridable
```

Comment:
Why do we need this when we already have `createCommonDepositFieldComponent` overridable definition?